draft-ietf-rats-corim.md

```diff
@@ -1816,6 +1864,13 @@ compared with the next entry of the Accepted Claims Set array.
 If all entries of the Reference Values array match a corresponding entry in the Accepted Claims Set array, then the `cryptokeys` Reference Value matches.
 Otherwise, `cryptokeys` does not match.
 
+##### Comparison for Integrity Registers {#sec-cmp-integrity-registers}
```

Comment:
```suggestion
##### Comparison for Integrity Registers {#sec-cmp-integrity-registers}
For each Integrity Register entry in the Reference Value, identified as: id => digests_{ref}, the Verifier will claim a "match" with Evidence only if the Evidence has an Integrity Register entry, identified as: id => digests_{ev} such that the `id` in Evidence and Reference Value both have the same type and value and the digests matches using the method defined in {{sec-cmp-digests}}.

If no entry is found, the Reference Value does not match.

Note that it is not required for all the Evidence entries in an Integrity Register array to be used during matching: Evidence could consist of a Target Environment's complete register space, while the Integrity Register array in Reference Values could consist of a subset of the Target Environment's register space.
```

Reply:
> id => digests_{ev}

This nomenclature is confusing as it doesn't refer to the actual CDDL. Substitute `id` with `integrity-register-id-type-choice` and ` => digests_{ev}` with ` => [ + digest ]` and it would be more clear. Alternatively, include the aliased terms in the CDDL definition. E.g.:
```
integrity-registers = {
  + id: integrity-register-id-type-choice => digests: [ + digest ]
}
```
Then it would be OK to use `id` and `digests`.

Reply:
> > the Verifier will claim a "match" with Evidence only if the Evidence has an Integrity Register entry
> 
> This wording is confusing because "Claim" has special meaning but it is used in the verb form. Other parts of the document use "match" in the verb form already. Maybe it's better to stay with this convention.
> 
> Nevertheless, the matching semantic is misrepresented by the sentence. The Verifier doesn't match evidence per se, as it accepts it simply because the (authenticated) Attester asserted it.
> 
> The Ref Val is accepted only if it is a subset of the Evidence.
> 
> The question that needs to be answered is whether or not the Ref Val subset requirements are applied to the IR array too. I think the answer should be yes.
> 
> The text I supplied seems to work in terms of describing applying the subset matching semantics to the IR array.

I agree we should use the term `match` only not mis-use the word claim.

Reply:
+1

Reply:
> The question that needs to be answered is whether or not the Ref Val subset requirements are applied to the IR array too. I think the answer should be yes.

Given @henkbirkholz https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/189#discussion_r1475724199 I am OK too.

> The text I supplied seems to work in terms of describing applying the subset matching semantics to the IR array.

LGTM

Reply:
I suggest we accept Andy's proposed text and merge this.  As suggested by Yogesh, we can track the refinement put forward by Ned in a separate PR.

Reply:
This works with me +1

Reply:
@nedmsmith : We have raised a new issue #199 to address your comments. We can discuss this in the incoming meeting. In the mean time, we intend to apply suggestions from @andrew-draper after clarification from @nedmsmith and acceptance from @thomas-fossati. I hope this is ok, with you.

Please reply with +1 if you agree so that we can un-block it today.

Reply:
+1